nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/awsfulltests.yml

```diff
@@ -0,0 +1,36 @@
+name: nf-core AWS full size tests
+# This workflow is triggered on push to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  push:
```

Comment:
Did we want to comment these out so that they never run, then add a TODO comment..?

Reply:
What I've done now, if you like the idea, is to have it on but it is running the small tests by default, and only when it is actively changed to the `test_full` profile, it will run the full tests. 

I've added a TODO to change from `-profile test` to `-profile test_full`

Reply:
I thought that would be neater, but just let me know if you would prefer to still have it commented out, I can change that of course!

Reply:
This is good 👍